.github/workflows/ci-app-examples.yml

```diff
@@ -58,12 +58,9 @@ jobs:
         if: ${{ matrix.requires == 'oldest' }}
         run: python .actions/assistant.py replace_oldest_ver
 
-      # Note: This uses an internal pip API and may not always work
-      # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
-      - name: Get pip cache
+      - name: Get pip cache dir
         id: pip-cache
-        run: |
-          python -c "from pip._internal.locations import USER_CACHE_DIR; print('::set-output name=dir::' + USER_CACHE_DIR)"
```

Comment:
I guess this is needed for multi-OS config...

Reply:
I remember pip didn't have `pip cache dir` at the time, but now it works on all operating systems as I see:
```
    path: c:\users\runneradmin\appdata\local\pip\cache
```
https://github.com/Lightning-AI/lightning/actions/runs/3368538476/jobs/5587210166#step:11:3